cfy_manager/components/validations.py

```diff
@@ -174,6 +175,31 @@ def _validate_inputs():
             )
 
 
+def _validate_host(host, input_name):
+    sock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+    try:
+        sock.connect((host, INTERNAL_REST_PORT))
```

Comment:
you can also make this shorter and ipv6-compat by using `socket.create_connection`, eg. like this:
```python
try:
  s = socket.create_connection((host, port))
except socket.error:
   # blabla
else:
  s.close()
```